body.tex

```diff
@@ -37,19 +38,18 @@ \section{Learning-based Approaches}
 \item starting off $(I_t,I_s)$ -- a.k.a end-to-end, TransiNet-style.
 \end{itemize}
 
-\footnote{The term 'detection' in the field of computer vision can be translated to localization+classification in the astrophysics' terminology}
-
 
-Pros and cons of the two approaches come in the following sections. However, the current decision is to implement and test the two in parallel. Therefore we shall attempt to bring the two implementations close to eachother -- ideally unified.
+Each of the approaches has its own cons and pros, and methods based on both will be implemented for LSST. In this tech-note we focus on details of the former, the real/bogus classifier, and the end-to-end TransiNet will be covered in a future note.
 
 
-\subsection{Basic Real/Bogus Classifier}
+\section{Basic Real/Bogus Classifier}
+\label{sec:basic}
 
 \begin{figure}[h]
   \centering
   \includegraphics[width=.6\textwidth]{material/rb-classifier}
   \includegraphics[width=.6\textwidth]{material/rb-classifier-mod}
-  \caption{Classical Real/bogus classifier (top) and the modified 3D classifier (bottom)}
+  \caption{Classical Real/bogus classifier (top, \cref{sec:basic}) and the modified 3D classifier (bottom,  \cref{sec:modified})}
```

Comment:
Citations to the "classical" example would be useful--e.g., the braai classifier uses all three images for input.